src/main/java/ch/njol/skript/registrations/Classes.java

```diff
@@ -292,7 +292,7 @@ private static void checkAllowClassInfoInteraction() {
 	}
 	
 	/**
-	 * Gets the class info of the given class or its closest registered superclass. This method will never return null unless <tt>c</tt> is null.
+	 * Gets the class info of the given class or it's closest registered superclass. This method will never return null unless <tt>c</tt> is null.
```

Comment:
```suggestion
	 * Gets the class info of the given class or its closest registered superclass. This method will never return null unless <tt>c</tt> is null.
```

The old one is correct